Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -18,8 +18,34 @@ function connected(jsn) {
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = {
+      'Content-Type': contentType
+    };
+    let inputHeaders = {};
+
+    if (typeof headers !== 'undefined') {
+      const headersArray = headers.split(/\n/);
+
+      for (let i = 0; i < headersArray.length; i += 1) {
+        const [headerItem, headerItemValue] = headersArray[i].split(';');
```

Comment:
We should ensure that we skip lines without the separator. Currently a black line will attempt to add a header with the invalid name `""`.